### PR TITLE
#164798243 Setup empty react project and redux 

### PR DESCRIPTION
**Pull request by anyatibrian**
#### What does this PR do?
Setup empty react project/redux
#### Description of Task to be completed?
- configure react and redux for the project
- setup redux thunk middleware
- setup redux store for the project
- setup the redux boilerplate for the entire project
- configure all the required dev dependencies 
#### How should this be manually tested?
After cloning the repo, cd into it and configure react dev environment 
- `$ npm install` that will install all the dev dependencies 
Run the server 
- `$ npm start`
#### What are the relevant pivotal tracker stories?
[#164798243](https://www.pivotaltracker.com/story/show/164798243)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import React from  'react';
+import ReactDom from 'react-dom';
+import App from './containers/app';
+ReactDom.render(<App/>, document.getElementById('App'));
```
Comment (Genza999) [inline src/index.js]: Hey @anyatibrian, great work. Kindly add a final new line at this file and all the other files.

**Review by patrickf949 — APPROVED** [review]
LGTM

File: src/index.js (hunk 1 above)
Comment (anyatibrian) [reply src/index.js]: which files@Genza cos those other files are just sample files to guide you

**Review by e-ian — APPROVED** [review]
LGTM

File: src/index.js (hunk 1 above)
Comment (Genza999) [reply src/index.js]: Hey @anyatibrian , src/components/app.js, babelrc, store.js, webpack.config.js

Comment (anyatibrian) [reply src/index.js]: yah @Genza999 its always good to follow the ASK format since it prevents you from giving abstract feedback.

**Review by Genza999 — APPROVED** [review]
Great work @anyatibrian. LGTM!

**Pull request merged**